pegasus/forms/pd_workshop_survey.rb

```diff
@@ -33,15 +33,30 @@ def self.normalize(data)
       result[:venue_feedback_s] = stripped data[:venue_feedback_s]
       result[:how_much_learned_s] = required_enum data, :how_much_learned_s
       result[:how_motivating_s] = required_enum data, :how_motivating_s
-      result[:how_clearly_presented_s] = required_enum data, :how_clearly_presented_s
-      result[:how_interesting_s] = required_enum data, :how_interesting_s
-      result[:how_often_given_feedback_s] = required_enum data, :how_often_given_feedback_s
-      result[:help_quality_s] = required_enum data, :help_quality_s
-      result[:how_comfortable_asking_questions_s] = required_enum data, :how_comfortable_asking_questions_s
-      result[:how_often_taught_new_things_s] = required_enum data, :how_often_taught_new_things_s
 
-      result[:things_facilitator_did_well_s] = stripped data[:things_facilitator_did_well_s]
-      result[:things_facilitator_could_improve_s] = stripped data[:things_facilitator_could_improve_s]
+      result[:who_facilitated_ss] = required data[:who_facilitated_ss]
+
+      unless result[:who_facilitated_ss].class == FieldError
```

Comment:
No, because of line 37; since this is a required field, leaving it blank counts as a FieldError